clang/test.cpp

```
@@ -0,0 +1,88 @@
+
+int Add(int a, int b) {
+    return a + b;
+}
+
+void leakyFunction() {
+    int* data = (int*)malloc(sizeof(int) * 10);
+    data[0] = 42;
+}
+
+int divide(int numerator, int denominator) {
+    return numerator / denominator;
```

Cppcheck (2.20.0)

[error] Division by zero.

(zerodiv)